packages/catalog-realm/components/map-render.gts

```diff
@@ -4,9 +4,11 @@ import { action } from '@ember/object';
 
```

Comment:
I can see this component is important. 
- config should have a type here even tho leaflet modifier might not accept type. 
- it will save you a lot of heartache to start grouping lat,lon into a single object 
- I dono how it fits into route. But we should have route as an array of points maybe. How does route handle the points?? There has to be some encoding